applications/luci-app-tailscale-community/htdocs/luci-static/resources/view/tailscale.js

```diff
@@ -0,0 +1,356 @@
+let callDoLogin = rpc.declare({ object: 'tailscale', method: 'do_login' });
+let callGetSubroutes = rpc.declare({ object: 'tailscale', method: 'get_subroutes' });
+let map;
+
+let tailscaleSettingsConf = [
+    [form.Flag, 'accept_routes', _('Accept Routes'), _('Allow accepting routes announced by other nodes.'), { rmempty: false }],
+    [form.Flag, 'advertise_exit_node', _('Advertise Exit Node'), _('Declare this device as an Exit Node.'), { rmempty: false }],
+    [form.Value, 'exit_node', _('Exit Node'), _('Specify an exit node. Leave it blank and it will not be used.'), { rmempty: true }],
+    [form.Flag, 'exit_node_allow_lan_access', _('Allow LAN Access'), _('When using the exit node, access to the local LAN is allowed.'), { rmempty: false }],
+    [form.Flag, 'shields_up', _('Shields Up'), _('When enabled, blocks all inbound connections from the Tailscale network.'), { rmempty: false }],
+    [form.Flag, 'ssh', _('Enable Tailscale SSH'), _('Allow connecting to this device through the SSH function of Tailscale.'), { rmempty: false }]
+];
+
+let daemonConf = [
+    [form.Value, 'daemon_mtu', _('Daemon MTU'), _('Set a custom MTU for the Tailscale daemon. Leave blank to use the default value.'), { datatype: 'uinteger', placeholder: '1280' }, { rmempty: false }],
+    [form.Flag, 'daemon_reduce_memory', _('Reduce Memory Usage'), _('Enabling this option can reduce memory usage, but it may sacrifice some performance (set GOGC=10).'), { rmempty: false }]
+];
+function setParams(o, params) {
+    if (!params) return; for (let key in params) {
+        let val = params[key]; if (key === 'values') {
+            for (let j = 0; j < val.length; j++) {
+                let args = val[j]; if (!Array.isArray(args))
+                    args = [args]; o.value.apply(o, args);
+            }
+        } else if (key === 'depends') {
+            if (!Array.isArray(val))
+                val = [val]; let deps = []; for (let j = 0; j < val.length; j++) {
+                    let d = {}; for (let vkey in val[j])
+                        d[vkey] = val[j][vkey]; for (let k = 0; k < o.deps.length; k++) { for (let dkey in o.deps[k]) { d[dkey] = o.deps[k][dkey]; } }
+                    deps.push(d);
+                }
+            o.deps = deps;
+        } else { o[key] = params[key]; }
+    }
+    if (params['datatype'] === 'bool') { o.enabled = 'true'; o.disabled = 'false'; }
+}
+function defTabOpts(s, t, opts, params) { for (let i = 0; i < opts.length; i++) { let opt = opts[i]; let o = s.taboption(t, opt[0], opt[1], opt[2], opt[3]); setParams(o, opt[4]); setParams(o, params); } }
+
+function getRunningStatus() {
+    return L.resolveDefault(callGetStatus(), { running: false }).then(function (res) {
+        return res;
+    });
+}
+
+function formatBytes(bytes) {
+    let bytes_num = parseInt(bytes, 10);
+    if (isNaN(bytes_num) || bytes_num === 0) return '-';
+    let k = 1024;

```

Comment:
This means one MB == 1048576 bytes, which is not correct. MB == Million Bytes. Or Mega. SI mandates that one million or one MB == 1000000 bytes. So `k = 1000`.

So either k = 1024 and the units are MiB (Mebibytes, base 2) or k = 1000 (SI, base 10) and the units are MB.